website/pages/guides/hcl/from-json-v1.mdx

```diff
@@ -168,13 +182,12 @@ There is soon going to be a PR to drop the `s` at the end of these fields.
 
 ### Deprecation
 
-The current layout of buildfiles will be supported until we and the community
-love the new format. Only then the v1 format will be carefully deprecated.
+As we become more confident in the new templates, we may begin to add new
```

Comment:
The diff is showing errors here. It looks like there is a missing closing bracket in the above JSON and a missing closing brace in the HCL example.